CMakeLists.txt

```diff
@@ -331,8 +331,12 @@ endif()
 #
 # Locate libsbml
 #
-find_package(LIBSBML REQUIRED)
-find_package(LIBNUML REQUIRED)
+if(NOT TARGET libsbml-static)
```

Comment:
I wonder if this should be here, as it could easily be in the individual find scripts, where we would not have to touch them for each project separately. Also I would probably choose the LIBSBML_LIBRARY_NAME rather than libbsml-static directly, as i know there are some people that would rather want to link against the shared library.